### ISSUE TITLE
Additional services

### ISSUE THREAD

**Issue by JakubMacoun** [shenxn/ha-dyson#16]
Would be nice to have additional services from current official integration. Currently, there are missing things like angles setting, flow direction etc.
Otherwise nice job!

**Comment by JakubMacoun** [comment]
Thank you, but I have one more question. What switches do you mean?

**Comment by shenxn** [comment]
You should be able to see entities `switch.*_night_mode`, `switch.*_continuous_monitoring`, and `switch.*_front_airflow` in Home Assistant.

**Comment by shenxn** [comment]
Try remove your config entry and re-add. Something may break during upgrading. If it still not working, see if you can find anything useful in log.

**Comment by shenxn** [comment]
Sorry for the confusion. I mean remove the device configuration from UI, and re-add. If not working, is your fan entity working?

**Comment by JakubMacoun** [comment]
I have removed Dyson Cloud and Dyson Local from integrations. Re-added Dyson Cloud and now I can't add Dyson Local - after the Host form submit, I get "Failed to find the device using discovery" error. Fan entity used to work before I removed the integration.

**Comment by Krocko** [comment]
I don‘t have the option `dyson_local.set_timer`. 
I have a pure cool link.

**Comment by JakubMacoun** [comment]
UPDATE: after removing all dyson integrations and restarting HA, I managed to re-add them. Switches for night mode and flow direction work well now.

**Comment by shenxn** [comment]
@Krocko Does Pure Cool Link support that? According to the code from official integration, it seems that `dyson.set_timer` is only added when there is a Pure Cool device.

**Comment by Krocko** [comment]
@shenxn Yes, I changed this in the fan.py and it is working for the pure cool link also.

**Comment by shenxn** [comment]
@Krocko Thanks for confirm. I'll add that soon.

**Comment by shenxn** [comment]
@Krocko I've released version 0.5.3. Now you should have the `set_timer` service.

**Comment by Pau1ey** [comment]
Is the dust info available for the Cool Pure Link fan?

**Comment by shenxn** [comment]
@Pau1ey You can find dust info from the`air_quality` entity. PM2.5 and PM10 values report the dust level.

**Comment by Krocko** [comment]
Thank you. It is working.
